DatadogInternal/Sources/MessageBus/FeatureMessageReceiver.swift

```diff
@@ -59,6 +59,6 @@ public struct CombinedFeatureMessageReceiver: FeatureMessageReceiver {
     ///   - core: An instance of the core from which the message is transmitted.
     /// - Returns: `true` if the message was processed by one of the receiver; `false` if it was ignored.
     public func receive(message: FeatureMessage, from core: DatadogCoreProtocol) -> Bool {
-        receivers.contains(where: { $0.receive(message: message, from: core) })
+        receivers.reduce(false) { $0 || $1.receive(message: message, from: core) }
```

Comment:
Previously, the `CombinedFeatureMessageReceiver` would stop forwarding the message after one of its receivers consumed a message. Now, it will send the message to all receivers and it returns `true` if at least one consumes it.

Reply:
blocking/ Changing this behaviour here should require change in [`FeatureMessageReceiverTests`](https://github.com/DataDog/dd-sdk-ios/blob/feature/v2/DatadogInternal/Tests/MessageBus/FeatureMessageReceiverTests.swift#L58-L70), but we don't do it in this PR 🤔💭. Looking at following test (that didn't require update on this branch), something is wrong - either the test makes bad assertion or we're not altering the logic the way it's described here.

https://github.com/DataDog/dd-sdk-ios/blob/ed77be78dcb516843927e961bdbfa001de9bebaf/DatadogInternal/Tests/MessageBus/FeatureMessageReceiverTests.swift#L58-L70